src/esp/bindings/SimBindings.cpp

```diff
@@ -54,6 +54,10 @@ void initSimBindings(py::module& m) {
       .def_readwrite(
           "allow_sliding", &SimulatorConfiguration::allowSliding,
           R"(Whether or not the agent can slide on NavMesh collisions.)")
+      .def_readwrite(
+          "include_static_objects_in_navmesh",
```

Comment:
I am sure there is a good reason, but why isn't this a navmesh setting? It requires a reconstruction of the Navmesh which we check by seeing if the new navmesh settings equals the old. Feels more naturally to live in there, no?

Reply:
Good point.
Currently, NavMeshSettings only includes recast-level parameters. The mesh joining triggered by this flag happens before the nav module is ever invoked, so it is really a Simaultor flag at the scene level. 

I think there could be a valid argument to add this to the NavMeshSettings and then pivot off of that in recompute_navmesh directly instead of treating it separately. That would be a breaking change across the stack.